src/dps-email-worker/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/dpsemailworker/DpsEmailConsumer.java

```diff
@@ -96,6 +92,43 @@ public void receiveMessage(DpsMetadata message) {
         } finally {
             MDC.remove(MdcConstants.MDC_TRANSACTION_ID_KEY);
         }
+
+    }
+
+    // Anything that hits the parking queue we can assume we can't process, so...
+    // 1. Move the email to the error folder in exchange
+    // 2. Delete the attached doc from the cache
+    @RabbitListener(queues = Keys.PARKING_QUEUE_NAME)
+    public void receiveParkedMessage(DpsMetadata message) {
```

Comment:
I would start with logging a fatal error for this message, the reason being that we need to track that we have a failure here, the messages should not make it to the parking lot unless exceptional circumstances, what do you think? in splunk we could setup alerts based on that message so someone could look into why a message was discard...

Reply:
Makes sense, I'll add it in!